core/templates/pages/exploration-player-page/templates/display-solution-modal.controller.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
This file is kind of hard to read... Can you add newlines where needed (like around functions), so that it is easy to read?

Reply:
Done.